articles/aks/faq.md

```diff
@@ -214,6 +214,10 @@ Except for the following two images, AKS images aren't required to run as root:
 - *mcr.microsoft.com/oss/kubernetes/coredns*
 - *mcr.microsoft.com/azuremonitor/containerinsights/ciprod*
 
+## I deleted an AKS cluster by mistake. Can I restore or recover it from any backups?
```

Comment:
should we soften the language and provide guidance for BCDR, storing the config as IaC so they can re-deploy, and so on?

https://docs.microsoft.com/en-us/azure/aks/operator-best-practices-multi-region#:~:text=%20Best%20practices%20for%20business%20continuity%20and%20disaster,3%20Enable%20geo-replication%20for%20container%20images...%20More%20

Reply:
Agree with Michael's original response that the proposed statement can be rephrased and reference the BCDR article. Because this PR remained in this state for so long without any follow-up by the submitter, I'm going to create a work item for this suggestion and close the PR.